grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServiceTest.java

```diff
@@ -119,4 +122,18 @@ public void missingMethod() throws Exception {
                            .set(HttpHeaderNames.CONTENT_LENGTH, "0"),
                 HttpData.of(new byte[] {})));
     }
+
+    @Test
+    public void pathMappings() throws Exception {
+       assertThat(grpcService.pathMappings())
+            .isEqualTo(
+                ImmutableSet.of(
```

Comment:
nit: Could use `containsExactly(...)` instead of `.isEqualTo(ImmutableSet.of(...))`